source/_integrations/zwave_js.markdown

```diff
@@ -959,10 +962,13 @@ Many users have reported issues with interference when the USB stick was directl
 
 ##### Enable Z-Wave JS logging
 
-1. Go to the Z-Wave integration panel: {% my integration badge domain="zwave_js" %}
-2. Select `Enable debug logging` on the left-hand side of the screen.
-
-The log level will be set to `debug` for the integration, library, and optionally the driver (if the driver log level is not already set to `verbose`, `debug`, or `silly`), and all Z-Wave JS logs will be added to the Home Assistant logs.
+1. Define the log level:
+   - By default, it is set to `info` for the integration, library, and optionally the driver (if the driver log level is not already set to `verbose`, `debug`, or `silly`), and all Z-Wave JS logs will be added to the Home Assistant logs.
+   - Go to the {% my integration domain="zwave_js" title="**Settings** > **Devices & services** > **Z-Wave**" %} and select **Configure**.
+   - On the Z-Wave configuration page, open the **Logs** tab.
+   - Choose your log level.
```

Comment:
This isn't needed to just set the log level to debug. It's enough to do step 2 and 3.

If we want to expand on the logging feature I think it should be a separate paragraph that explains that you can customize the logging level of the driver. This is what's done from the logging panel in the config panel of the Z-Wave integration.

If the user just wants to capture debug logs for troubleshooting (the most common case) it's enough to follow the previous description.